SDK/New Group/Models/SettingsObject.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  SettingsObject.swift
+//  iOS-SDK
+//
+//  Created by Alberto Huerdo on 12/17/18.
+//  Copyright © 2018 SchedJoules. All rights reserved.
+//
+
+import Foundation
+import SchedJoulesApiClient
+
+struct SettingsObject: CodedOption {
+    
+    enum CodingKeys: String, CodingKey {
+        case code = "code"
+        case name = "name"
+        case path = "path"
+    }
+    
+    
+    var name: String
```

Comment:
To me it looks like it's perfectly possible to make this class immutable and turn all these into `let`s. Have you considered this?

Reply:
Yes, I usually start all properties as immutable, but there are two reasons I change them:
1. when the user hasn't chosen a country or a language we don't have all the properties, which in swift causes an error and we would need to add dummy values.
2. In SettingsManager we asking the value to the code based on the type of the object, if we turn them into constants we wouldn't be able to do it.